### DISCUSSION TITLE
templ version check failed

### DISCUSSION THREAD

**Discussion by fredrikbergqvist** [a-h/templ#460] [Q&A] [answered]
Hello!
I'm new to Go, so this may be super obvious, but I must ask :)

When I run `templ generate --watch` I get spammed with messages about my templ cli is old:

```
(!) templ version check failed: generator v0.2.501 is older than templ version v0.2.513 found in go.mod file, consider upgrading templ CLI
```

On my mac I just ran `go get -u github.com/a-h/templ` and it updated (at least no more messages) but on my windows box nothing happens. 

Does anyone know what could be wrong?

**Comment by swetjen** [comment, marked as answer]
You're probably running an outdated version of the CLI tool, not the package dependency.  To confirm, try this from your command line:

`$ templ version`

It'll probably show `v0.2.501`.  If so, you can fix this by updating the CLI. Assuming you've installed templ using go, the command to update templ is:

`$ go install github.com/a-h/templ/cmd/templ@latest`.

**Reply by fredrikbergqvist** [reply]
Worked like a charm, many thanks!
